wiki/Gameplay/Game_modifier/Target_Practice/en.md

```diff
@@ -26,11 +26,17 @@ tags:
 
 *Notice: Enabling the Target Practice mod will result in an unranked play.*
 
-The **Target Practice** mod is an experimental [game modifier](/wiki/Gameplay/Game_modifier) that removes all the mapped [hit objects](/wiki/Gameplay/Hit_object) and replaces them with a simplified and consistent set of "targets".
+The **Target Practice** mod is an experimental [game modifier](/wiki/Gameplay/Game_modifier) for [osu!](/wiki/Game_mode/osu!), which removes all the mapped [hit objects](/wiki/Gameplay/Hit_object) and replaces them with a simplified and consistent set of "targets". Due to this mod's nature, The mod is used mostly for fun, but can also be helpful to players that want to improve themselves, for they can use it to practice maintaining a consistent tempo and improving hit accuracy.
 
-During a play with the Target Practice mod enabled, players will be presented with a set of "targets" that slowly increase in size and brightness. The player is then expected to tap or click the targets when they are at full brightness and size and the time is right. A consistent metronome will be played in the background to indicate to players when they should tap the target.
+During a play with the Target Practice mod enabled, the health bar and accuracy will not be shown. The player will be presented with a set of "targets" that slowly increase in size and brightness. The player is then expected to tap or click the targets when they are at full brightness and size and the time is right. A consistent metronome will be played in the background to indicate to players when they should tap the target.
```

Comment:
> The player is then expected to tap or click the targets when they are at full brightness and size

This doesn't sound right. It should all be about timing, there's no way a user can gauge the full brightness of a target.